src/app/pages/wallets/my-wallets/pages-wallets-my-wallets.component.ts

```diff
@@ -13,10 +15,12 @@ export class PagesWalletsMyWalletsComponent implements OnInit {
     isLoading: true,
     data: null,
     hasError: false,
+    nowEditWallet: 0,
```

Comment:
Instead of this whole magic with `nowEditWallet` for creating IDs for wallet etc. You can just use `getRandomNumber` from `common.utils.random.ts` and create some random ID for now (you can generate for example random number between 1000 and 10000). This is anyway temporary and as soon as backend is implemented we won't need this.